### PR TITLE
[CI] -[LIVE-10166] - Improve CI readability on `build-desktop.yml`

### PR DESCRIPTION
**Pull request by sshmaxime**
### 📝 Description

This PR aims at improving the CI jobs and workflows readability for `build-desktop.yml`.

### ❓ Context

- **JIRA or GitHub link**: [LIVE-10166] 

### ✅ Checklist

Pull Requests must pass the CI and be code reviewed. Set as Draft if the PR is not ready.

- [ ] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - 

---

### 🧐 Checklist for the PR Reviewers

- [ ] **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- [ ] **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- [ ] **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- [ ] **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- [ ] **Any new dependencies** have been justified and documented.
- [ ] **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-10166]: https://ledgerhq.atlassian.net/browse/LIVE-10166?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/ledger-live-desktop/src/renderer/actions/accounts.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { AccountComparator } from "@ledgerhq/live-common/account/index";
 import { implicitMigration } from "@ledgerhq/live-common/migrations/accounts";
 import { getKey } from "~/renderer/storage";
 
+//la
```
Comment (valpinkman) [inline apps/ledger-live-desktop/src/renderer/actions/accounts.ts:7]: remove

**Comment by aboissiere-ledger** [comment]
@sshmaxime Please add the associated JIRA ticket in the description. Thanks

File: .changeset/poor-birds-know.md
Comment (valpinkman) [inline .changeset/poor-birds-know.md:1]: Honestly, not really needed as these are not public facing packages. But ok 👍

**Review by valpinkman — COMMENTED** [review]
Remove the directory `desktop` for the new action and we are golden 👌

**Review by valpinkman — APPROVED** [review]

**Pull request merged**